### DISCUSSION TITLE
فعال کردن برای تلگرام

### DISCUSSION THREAD

**Discussion by khebar** [GFW-knocker/gfw_resist_tls_proxy#97] [Q&A]
سلام میشه این **تیزافزار**😁 رو برای تلگرام هم فعال کرد؟

**Comment by GFW-knocker** [comment]
بعیده
باید ایپی فیلتر نشده تلگرام  پیدا کنیم 
یوتیوب هم که بازه بخاطر اینه که ایپی هاش با سایت گوگل مشترکه و اینا میخوان گوگل باز باشه
دنبال یه راه اصولی هستیم که بریزه بهم کاسه کوزشونو
